setting_up_firely_server/configuration/plugins/available_plugins.rst

```diff
@@ -44,6 +40,13 @@ Infrastructural plugins
 :Order: 130
 :Description: Registers an implementation for the ``ISerializationService`` and ``ISerializationSupport`` interfaces and actual serializers and parsers for JSON and XML.
 
+.. _vonk_plugins_core_services:
```

Comment:
Should this not be listed as part of the Terminology Service paragraph?

Reply:
I would rather want to keep it here. For me this configuration is all about core infrastructure.